cookies/templates/define_resource_content_region.html

```diff
@@ -0,0 +1,127 @@
+{% extends "base.html" %}
+{% load i18n %}
+{% load addcss %}
+{% load preview %}
+
+{% load breadcrumbs %}
+{% block breadcrumbs %}
+<div class="breadcrumbs">
+    <a href="{% url 'index' %}">{% trans 'Home' %}</a>
+    {% breadcrumb_url 'Resources' 'resources' %}
+    {% breadcrumb_url resource.name 'resource' resource.id  %}
+    {% breadcrumb_url 'Content Regions' %}
+    {% breadcrumb_url 'Define' 'define-resource-content-region' resource.id  %}
+</div>
+{% endblock %}
+
+{% block content %}
+
+<div class="row">
+    <div class="col-md-6">
```

Comment:
Is there a reason you're not using the whole width of the page? Otherwise looks great!

Reply:
All the form elements get a little too wide for reasonably higher resolutions. In all the instances I have seen, the preview text lines are never that long. I will still go ahead and change the class to `col-12 col-md-8` to use maximum width for lower resolutions and 8 columns (instead of 6) for higher resolutions.

Will that be okay?